src/modules/simulator/simulator_mavlink.cpp

```diff
@@ -735,7 +731,7 @@ void Simulator::pollForMAVLinkMessages(bool publish)
 	struct sched_param param;
 	(void)pthread_attr_getschedparam(&sender_thread_attr, &param);
 
-	param.sched_priority = SCHED_PRIORITY_DEFAULT;
+	param.sched_priority = SCHED_PRIORITY_ACTUATOR_OUTPUTS + 1;
```

Comment:
Why? And why is this part of this pull request?

Reply:
Comment added.

Reply:
Btw. as far as I know these priorities are ignored unless you run px4 as root.